extensions/guacamole-auth-radius/src/main/resources/guac-manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+
+    "guacamoleVersion" : "0.9.14",
+
+    "name"      : "RADIUS Authentication Backend",
+    "namespace" : "radius",
+
+    "authProviders" : [
+        "org.apache.guacamole.auth.radius.RadiusAuthenticationProvider"
+    ],
+
+    "translations" : [
+        "translations/en.json"
+    ],
+
+    "js" : [
+        "radius.min.js"
+    ],
+
+    "resources" : {
+        "templates/radiusChallengeResponseField.html" : "text/html"
```

Comment:
It looks like this has been renamed to "templates/radiusResponseField.html". You'll need to add "templates/radiusStateField.html", as well.

Reply:
Updated the one, and added the other.